app/src/main/java/com/example/voyagerx/repository/LaunchRepository.kt

```diff
@@ -0,0 +1,39 @@
+package com.example.voyagerx.repository
+
+import apolloClient
+import com.example.rocketreserver.LaunchListQuery
+
+class LaunchRepository {
```

Comment:
Oh like a companion object? That's a cool idea, I wonder if that would be more efficient. Does the presenter get destroyed and recreated along with the fragment its attached to? That would be a good question to bring up during today's standup.

I don't think it would be necessary for this project because I think only the landing page's presenter will need access to this repository. The other pages are only reading from the Room database.

Reply:
I was able to convert the Repository to a singleton object by using the `object` keyword. Thank you for the suggestion! Now any presenter can use the repository by calling `LaunchRepository.getLaunches()`.

Singleton in Kotlin doc: https://www.journaldev.com/18662/kotlin-singleton-companion-object

Change is here: 3a2be0a346b45c194fef35321a7ca18ff53926f7